demisto_sdk/commands/content_graph/objects/base_content.py

```diff
@@ -316,6 +311,46 @@ def from_path(
     def match(_dict: dict, path: Path) -> bool:
         pass
 
+    @property
+    def related_content(self) -> dict:
+        if not self.related_content_dict:
+            self.related_content_dict = self.get_related_content()
+            if self.old_base_content_object:
+                git_util = GitUtil()
```

Comment:
no need to load the git-util each time, but only once, see what i did in the `File` object

Reply:
We know this will be called once so I'm not sure it's relevant.